### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Sirpyerre**
Update README with Linux build instructions and dependencies for hajimehoshi/ebiten package.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,7 +56,30 @@ This will get you the binary with an icon and it does not show a window on the t
...
+sudo apt install libc6-dev libgl1-mesa-dev libxcursor-dev libxi-dev libxinerama-dev libxrandr-dev libxxf86vm-dev libasound2-dev pkg-config
+```
+
+For more information, refer to the official [hajimehoshi/ebiten documentation](https://ebitengine.org/en/documents/install.html?os=linux).
+
+Install Go dependencies:
+
+```bash
+go mod tidy
+go mod vendor
```
Comment (phrozen) [inline README.md]: The most common way to install dependencies is just plain `go get`, I want to be consistent

Comment (phrozen) [reply README.md]: `go mod vendor ` is a preference as is not necessary to build the app.

File: README.md [hunk 2]
```diff
@@ -56,7 +56,30 @@ This will get you the binary with an icon and it does not show a window on the t
...
+```
+
+Build the binary:
+
+```bash
+go build -o digital-clock
```
Comment (phrozen) [inline README.md]: just `go build` does the same, as it takes the package name as the output name. Maybe we can omit all the different Go commands and instead try to add static flags?

File: README.md [hunk 3]
```diff
@@ -56,7 +56,30 @@ This will get you the binary with an icon and it does not show a window on the t
...
+```
+
+## Usage
+
+Simply run the binary to display the clock on your screen. The clock will remain on top of other windows and can be moved by dragging.
```
Comment (phrozen) [inline README.md:81]: I'm going to move usage to the top. Thanks for the reminder

**Review by phrozen — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Sirpyerre) [reply README.md]: Ok, I changet it!

File: README.md (hunk 2 above)
Comment (Sirpyerre) [reply README.md]: I remove

**Review by phrozen — APPROVED** [review]

**Pull request merged**